src/time-map.scm

```diff
@@ -0,0 +1,20 @@
+(use-modules ((rnrs) :version(6)))
```

Comment:
please do not use rnrs if at all possible. There are much better ways to do things than to use that.